src/components/UserInterface/UserDisplay/UserDisplay.js

```diff
@@ -3,12 +3,33 @@ import UserAccount from "../../SearchResults/UserAccount/UserAccount";
 import UserContact from "../../SearchResults/UserContact/UserContact";
 import styles from "./UserDisplay.module.css";
 
-const UserDisplay = ({ icons, userData: { profile, account, contact } }) => {
+const UserDisplay = ({ userData }) => {
+  const {
+    avatar_url,
+    name,
+    login,
+    created_at,
+    bio,
+    public_repos,
+    followers,
+    following,
+    location,
+    twitter_username,
+    blog,
+    company,
+  } = userData;
+
+  const displayData = {
+    profile: [avatar_url, name, login, created_at, bio],
+    account: [public_repos, followers, following],
+    contact: [location, twitter_username, blog, company],
+  };
+  console.log("data: ", userData);
   return (
     <div className={styles.userDisplay}>
-      <UserInfo userInfo={{ ...profile }} />
-      <UserAccount accountInfo={{ ...account }} />
-      <UserContact icons={icons} contactInfo={{ ...contact }} />
+      <UserInfo userInfo={{ ...displayData.profile }} />
+      <UserAccount accountInfo={{ ...displayData.account }} />
```

Comment:
For `UserAccount` and `UserInfo` you are actually just passing down an object prop called `userInfo` or `accountInfo` but you are destructuring at the component level using the prop properties. For that to work, you can pass down props like this:
```suggestion
      <UserAccount {..displayData.account } />
```

With what you have now, searching a user doesn't trigger any UI changes because the values you are targeting at the component level will always be `undefined`

Reply:
I'm sorry, I am not at all clear on this either. Can you shed more light on this?